pkg/service/model_build_load_balancer.go

```diff
@@ -232,42 +232,94 @@ func (t *defaultModelBuildTask) resolveLoadBalancerSubnets(ctx context.Context,
 }
 
 func (t *defaultModelBuildTask) buildLoadBalancerAttributes(_ context.Context) ([]elbv2model.LoadBalancerAttribute, error) {
-	var attrs []elbv2model.LoadBalancerAttribute
-	accessLogEnabled := t.defaultAccessLogS3Enabled
-	bucketName := t.defaultAccessLogsS3Bucket
-	bucketPrefix := t.defaultAccessLogsS3Prefix
-	if _, err := t.annotationParser.ParseBoolAnnotation(annotations.SvcLBSuffixAccessLogEnabled, &accessLogEnabled, t.service.Annotations); err != nil {
+	defaultAttribs := t.getDefaultAnnotationLbAttributes()
+	configMapAttribs, err := t.getConfigMapLbAttributes()
```

Comment:
lets rename to getLoadBalancerAttributes instead. Config map is a bit confusing since there is no configmap resource involved. Also, loadBalancerAttributes instead of configMapAttributes.
nit: loadBalancerAttributes instead of configMapAttribs